semantic_conventions/metrics/http.yaml

```diff
@@ -1,17 +1,60 @@
 groups:
-  - id: metric.http.server.duration
-    type: metric
-    metric_name: http.server.duration
-    brief: "Measures the duration of inbound HTTP requests."
-    instrument: histogram
-    unit: "s"
+  - id: metric_attributes.http.server
+    type: attribute_group
+    brief: 'HTTP server attributes'
     extends: attributes.http.server
+    attributes:
+      - ref: server.address
+        brief: >
+          Name of the local HTTP server that received the request.
+        requirement_level: opt_in
+        note: |
+          Determined by using the first of the following that applies
+
+          - The [primary server name](/specification/trace/semantic_conventions/http.md#http-server-definitions) of the matched virtual host. MUST only
+            include host identifier.
+          - Host identifier of the [request target](https://www.rfc-editor.org/rfc/rfc9110.html#target.resource)
+            if it's sent in absolute-form.
+          - Host identifier of the `Host` header
+
+          SHOULD NOT be set if only IP address is available and capturing name would require a reverse DNS lookup.
```

Comment:
We need to duplicate this because server.address is not HTTP specific, and we don't have a way to reference a particular overriden ref, I guess? Seems like a potential tooling improvement...

Reply:
I created an issue to track it in the build-tools repo: https://github.com/open-telemetry/build-tools/issues/183